packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -206,9 +206,9 @@ def __init__(
 
         # Title case tool names to match third party demonstration data
         self.tools = [
-            create_tool(self.search, "Search"),
-            create_tool(self.construct_lookup_list, "Lookup"),
-            create_tool(self.submit_answer, "SubmitAnswer"),
+            create_tool(self.search, "search"),
```

Comment:
If we want this, let's use straight up `Tool.from_function`, and remove `create_tool` usage here